packages/opentelemetry/src/metrics.ts

```diff
@@ -91,3 +91,73 @@ export async function instrumentedWithMetrics<T>(
     histogram.record(performance.now() - start);
   }
 }
+
+export interface CreateObservableRangeGaugeOptions extends MetricOptions {
+  interval: number;
+}
+
+/**
+ * Creates a set of gauges that track the min, max, and average of a value over
+ * time. The value is observed on a regular interval.
+ *
+ * The provided {@link name} is used as the base name for the three gauges. The
+ * names of the individual gauges are:
+ *
+ * - `${name}.min`
+ * - `${name}.max`
+ * - `${name}.avg`
+ */
+export async function createObservableRangeGauge(
```

Comment:
Bikeshedding: any ideas for a better name?

Reply:
These both sound fine to me (or at least, equally unclear 😄) and I can't think of anything better.

Reply:
`observeValueStats` maybe? 🤔

Reply:
Honestly I'm fine with your current name in the code.